sklearn/ensemble/tests/test_bagging.py

```diff
@@ -471,8 +471,6 @@ def test_error():
 
 def test_parallel_classification():
     # Check parallel classification.
-    rng = check_random_state(0)
-
     # Classification
     X_train, X_test, y_train, y_test = train_test_split(
         iris.data, iris.target, random_state=rng
```

Comment:
Let's keep this test independent of the other test execution order:

```suggestion
        iris.data, iris.target, random_state=0
```

Reply:
Done